lib/rubocop/cop/lint/unneeded_require_statement.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # Checks for unnecessary require statement.
```

Comment:
I fixed the same mistake as https://github.com/bbatsov/rubocop/pull/4764#discussion_r140652474.